providers/clickatell/test.ts

```diff
@@ -0,0 +1,10 @@
+import { ClickatellSmsProvider } from './src/index';
+
+const provider = new ClickatellSmsProvider({
+  apiKey: 'otbjTKxDRlGxGlA6wiSCKQ==',
```

Comment:
I think you exposed your key by mistake

Reply:
@jainpawan21 this has been removed. It was a file I used for testing the functionality